app.py

```diff
@@ -113,6 +134,51 @@ class MaxContentLengthWarning(APIWarnings):
 app.config["MAX_CONTENT_LENGTH"] = MAX_CONTENT_LENGTH_MEGABYTES * 1024 * 1024
 
 
+@app.before_serving
+async def before_serving():
+    try:
+        # Check: do environment variables exist?
+        if CONNECTION_STRING is None:
+            raise ServerError("Missing environment variable: NACHET_AZURE_STORAGE_CONNECTION_STRING")
+
+        if FERNET_KEY is None:
+            raise ServerError("Missing environment variable: FERNET_KEY")
+
+        if PIPELINE_VERSION is None:
+            raise ServerError("Missing environment variable: PIPELINE_VERSION")
+
+        if PIPELINE_BLOB_NAME is None:
+            raise ServerError("Missing environment variable: PIPELINE_BLOB_NAME")
+
+        if NACHET_DATA is None:
+            raise ServerError("Missing environment variable: NACHET_DATA")
+
+        # Check: are environment variables correct?
+        if not bool(re.match(connection_string_regex, CONNECTION_STRING)):
+            raise ServerError("Incorrect environment variable: NACHET_AZURE_STORAGE_CONNECTION_STRING")
+
+        if not bool(re.match(pipeline_version_regex, PIPELINE_VERSION)):
+            raise ServerError("Incorrect environment variable: PIPELINE_VERSION")
+
+        CACHE["seeds"] = await fetch_json(NACHET_DATA, "seeds", "seeds/all.json")
```

Comment:
what does this json contain?

Reply:
Here is the link [nachet-data/seed/all.,json](https://github.com/ai-cfia/nachet-data/blob/main/seeds/all.json),  it is data from all the seeds collected by Jolan.